source/documentation/fuseki2/fuseki-configuration.md

```diff
@@ -124,9 +124,71 @@ An in-memory dataset, with data in the default graph taken from a local file.
         ## tdb:unionDefaultGraph true ;
          .
 
+#### TDB2
+
+    <#dataset> rdf:type      tdb:DatasetTDB2 ;
+        tdb:location "/some/path" ;
+        # this can also have unionDefaultGraph, etc.
```

Comment:
Maybe use the same example above from TDB? Copying that part with the timeout (quite sure it's the same syntax, there's another example in [the docs](https://jena.apache.org/documentation/fuseki2/fuseki-config-endpoint.html)).

Same for `unionDefaultGraph`. Maybe just leave it commented out.